rs/ledger_suite/icrc1/ledger/src/main.rs

```diff
@@ -146,6 +156,16 @@ fn pre_upgrade() {
     });
 }
 
+#[cfg(not(feature = "low-upgrade-instruction-limits"))]
+const MAX_INSTRUCTIONS_PER_UPGRADE: u64 = 199_950_000_000;
```

Comment:
I took the limits from https://internetcomputer.org/docs/current/developer-docs/smart-contracts/maintain/resource-limits. I think they now increased (timer seems to have a limit of 40B), so we might also increase them. I left a 50M margin - I noticed in test that with 500k limit it sometimes executed ~900k instructions, so 50M should be safe.

Reply:
Oh, and upgrade is now 300B.